demo/app/controllers/hotdog.js

```diff
@@ -12,7 +14,9 @@ class hotdog {
     }
 
     wutang(req, res) {
-        res.send('wutang son!');
+        Database.table("test").find(1).then(function(data) {
+            res.send('wutang son! Also,<br />' + JSON.stringify(data));
```

Comment:
Thank you for maintaining the popular debugging `wutang` methodology and not overwriting the initial debug message. Also I love this syntax. :)